src/Deprecated/Engine/Engine/EngineLoggingServices.cs

```diff
@@ -21,12 +21,12 @@ namespace Microsoft.Build.BuildEngine
     /// services for tasks. This class allows for better encapsulation by making it clear when only
     /// logging services are needed and not the rest of the engine data and methods. This class allows
     /// us to provide different implementations of logging services for engine components that are
-    /// either in-proc (engine process) or out-of-proc (node process).
+    /// either in-proc (engine process) or out-of-proc (node process). 
```

Comment:
Nit: don't add this whitespace

Reply:
Not sure how this happened. This is deprecated type :D Fixed.